OrbitLinuxTracing/PerfEvent.h

```diff
@@ -131,10 +133,31 @@ class LostPerfEvent : public PerfEvent {
   uint64_t GetNumLost() const { return ring_buffer_record.lost; }
 };
 
-template <typename perf_record_t>
+struct __attribute__((__packed__)) dynamically_sized_perf_event_stack_sample {
```

Comment:
What's the convention for typenames we want to follow? Some here a PerfEvent, some are perf_event.

Reply:
So, perf_event events correspond to ring buffer records, i.e. what we read from the ring buffers. PerfEvent are our classes that carry events. Most PerfEvent classes contain a corresponding perf_event record but that's for simplicity.